src/modules/Dashboard/features/Applet/Schedule/Legend/SearchPopup/SearchPopup.tsx

```diff
@@ -133,8 +134,8 @@ export const SearchPopup = ({
             </Box>
 
             <StyledItemsContainer>
-              {respondentsItems &&
-                filteredRespondents(respondentsItems)?.map((item) => {
+              {filteredRespondents &&
+                filteredRespondents.map((item) => {
```

Comment:
maybe could be just {filteredRespondents?.map(... ?

Reply:
fixed